reana_workflow_controller/config.py

```diff
@@ -15,6 +15,8 @@
 
 from reana_workflow_controller.version import __version__
 
+from reana_db.config import SQLALCHEMY_DATABASE_URI
+
```

Comment:
Imported but not used?

Reply:
It's used to pick `REANA_SQLALCHEMY_DATABASE_URI` env var from `run-tests.sh` but I've moved it to conftest.py